presto-main/src/main/java/com/facebook/presto/util/GraphvizPrinter.java

```diff
@@ -185,6 +185,22 @@ public static String printDistributed(SubPlan plan, Session session, FunctionAnd
 
         return output.toString();
     }
+    public static String printDistributedFromFragments(List<PlanFragment> allFragments, Session session, FunctionAndTypeManager functionAndTypeManager)
```

Comment:
Let's add a test in [TestGraphvizPrinter.java](https://github.com/prestodb/presto/blob/7d167d5fe7c1128ef95eb2fbee0b2ac3e80d6337/presto-main/src/test/java/com/facebook/presto/util/TestGraphvizPrinter.java) to exercise this as well